haproxy/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM haproxy:1.7
+FROM haproxy:1.7-alpine
+
+COPY docker-entrypoint2.sh reload-haproxy.sh /
+ENTRYPOINT ["/docker-entrypoint2.sh"]
+CMD ["haproxy", "-f", "/usr/local/etc/haproxy/haproxy.cfg"]
```

Comment:
I wonder if we need the `CMD` since the base image already has it?

Reply:
it did not work without it on my machine 😞